Makefile

```diff
@@ -193,3 +193,26 @@ send-tokens-to-ethereum-mainnet:
 		--account $(ACCOUNT) \
 		--broadcast \
 		-vvv
+
+#
+# Admin role transfer operations
+#
+
+# Transfer admin role for a single chain
+transfer-admin-single-chain: # CHAIN, RPC_URL, NEW_ADMIN
```

Comment:
Why specify `single-chain` here?

Reply:
`begin-default-admin-transfer` since, practically, it does not make the transfer, it only inits it.